modules/logging-install-es-operator.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-deploying.adoc
+
+:_content-type: PROCEDURE>=
+[id="logging-install-es-operator_{context}"]
+= Installing the OpenShift Elasticsearch Operator by using the web console
+
+The OpenShift Elasticsearch Operator creates and manages the Elasticsearch cluster used by OpenShift Logging.
+
+.Prerequisites
+
+* Elasticsearch is a memory-intensive application. Each Elasticsearch node needs at least 16GB of memory for both memory requests and limits, unless you specify otherwise in the `ClusterLogging` custom resource.
++
+The initial set of {product-title} nodes might not be large enough to support the Elasticsearch cluster. You must add additional nodes to the {product-title} cluster to run with the recommended or higher memory, up to a maximum of 64GB for each Elasticsearch node.
++
+Elasticsearch nodes can operate with a lower memory setting, though this is not recommended for production environments.
+
+* Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node
+requires its own storage volume.
++
+[NOTE]
+====
+If you use a local volume for persistent storage, do not use a raw block volume, which is described with `volumeMode: block` in the `LocalVolume` object. Elasticsearch cannot use raw block volumes.
+====
+
+.Procedure
+
+. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+. Choose  *OpenShift Elasticsearch Operator* from the list of available Operators, and click *Install*.
```

Comment:
[ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#c) says to use _click_ or _select_ and to avoid _choose_ for UI interactions.